halo2_gadgets/src/vk_ecc_chip

```diff
@@ -0,0 +1,9915 @@
+PinnedVerificationKey {
```

Comment:
follow the same file name convention to indicate this is a test case:
`vk_ecc_chip_0`
Same for sinsemilla